src/NetMQ/NetMQSocket.cs

```diff
@@ -343,18 +343,10 @@ internal void InvokeEvents(object sender, PollEvents events)
             m_socketEventArgs.Init(events);
 
             if (events.HasIn())
-            {
-                var temp = m_receiveReady;
-                if (temp != null)
-                    temp(sender, m_socketEventArgs);
-            }
+                m_receiveReady?.Invoke(sender, m_socketEventArgs);
```

Comment:
Yep, this is a safe transformation thankfully. Makes the code read a lot more nicely.